src/github.js

```diff
@@ -964,6 +964,22 @@
                body: comment
             }, cb);
          };
+         this.edit = function (issue,options,cb) {
+             if (typeof issue === 'object') {
```

Comment:
In this case the first argument should be a number, it shouldn't be derived from the options.